src/tests/cart_memory/write.rs

```diff
@@ -195,19 +195,19 @@ impl Test for DecayAfterSomeClockCycles {
         match (*value).downcast_ref::<u32>() {
             Some(&64) => {
                 const INSTRUCTION: u32 = Assembler::make_loadstore(Opcode::SD, 4, 0, 2);
-                DecayAfterSomeClockCycles::test::<INSTRUCTION, 20, 100>()?;
+                DecayAfterSomeClockCycles::test::<INSTRUCTION, 10, 100>()?;
```

Comment:
unrelated to this patch, but I see hardware errors from time to time, so lowering the threshold here